lib/release_notes_bot_web/controllers/webhook_controller.ex

```diff
@@ -5,7 +5,7 @@ defmodule ReleaseNotesBotWeb.WebhookController do
   use ReleaseNotesBotWeb, :controller
   alias ReleaseNotesBot.{Projects, WebhookEvents, Repositories, Clients, Channels, Persists}
 
-  @release_actions ["published", "edited", "deleted"]
+  @release_actions ["published", "deleted"]
```

Comment:
this seems straightforward. where can i read more about this though? i'm trying to find someplace i can read about `@release_actions`

Reply:
oh i think i found where we are processing these actions a little further down.

Reply:
so by removing `"edited"` from this array, we are effectively ignoring it as we process events?

Reply:
Correct! I will add docs about listening to and ignoring release actions.